Dockerfile.junit

```diff
@@ -2,7 +2,7 @@ FROM golang:1.17-alpine
 MAINTAINER CyberArk Software Ltd.
 LABEL id="conjur-authn-k8s-client-junit-processor"
 
-WORKDIR /test
+WORKDIR /conjur-authn-k8s-client/test
```

Comment:
Why is this change needed ?

Reply:
This Dockerfile is used to convert the standard `go test` output to formats consumable by Cobertura and CodeClimate. That said, `gocov-xml` was failing when it didn't have access to the project's source files:

```
error: can't find "main.go": no required module provides package github.com/cyberark/conjur-authn-k8s-client/cmd/authenticator: go.mod file not found in current directory or any parent directory; see 'go help modules'
```